opencti-platform/opencti-front/src/components/ItemSeverity.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { FunctionComponent } from 'react';
+import Chip from '@mui/material/Chip';
+import makeStyles from '@mui/styles/makeStyles';
+
+const useStyles = makeStyles(() => ({
+  chip: {
+    fontSize: 12,
+    height: 25,
+    marginRight: 7,
+    textTransform: 'uppercase',
+    borderRadius: '0',
+    width: 80,
+  },
+  chipInList: {
+    fontSize: 12,
+    height: 20,
+    float: 'left',
+    textTransform: 'uppercase',
+    borderRadius: '0',
+    width: 80,
+  },
+}));
+
+const inlineStyles = {
+  green: {
+    backgroundColor: 'rgba(76, 175, 80, 0.08)',
+    color: '#4caf50',
+  },
+  blue: {
+    backgroundColor: 'rgba(92, 123, 245, 0.08)',
+    color: '#5c7bf5',
+  },
+  red: {
+    backgroundColor: 'rgba(244, 67, 54, 0.08)',
+    color: '#f44336',
+  },
+  orange: {
+    backgroundColor: 'rgba(255, 152, 0, 0.08)',
+    color: '#ff9800',
+  },
+  blueGrey: {
+    backgroundColor: 'rgba(96, 125, 139, 0.08)',
+    color: '#607d8b',
+    fontStyle: 'italic',
+  },
+};
+
+interface ItemSeverityProps {
+  label: string;
+  severity?: string;
+  variant?: 'inList';
+}
+
+const ItemSeverity: FunctionComponent<ItemSeverityProps> = ({
+  label,
+  severity,
+  variant,
+}) => {
+  const classes = useStyles();
+  const style = variant === 'inList' ? classes.chipInList : classes.chip;
+  switch (severity) {
```

Comment:
We can potentially reduce the number of lines by making a switch only on the style part and return :
<Chip
          classes={{ root: style }}
          style={COMPUTE_STYLE}
          label={label}
        />

Reply:
Done